chain-vote/src/lib.rs

```diff
@@ -17,8 +17,8 @@ pub mod debug {
     pub mod gang {
         pub use crate::gang::*;
     }
-    pub mod shvzk {
-        pub use crate::shvzk::*;
+    pub mod private_voting {
```

Comment:
much prefer the old name personally ¯\_(ツ)_/¯

Reply:
I don't have a strong feeling about `private_voting`, but special honest verifier zero knowledge (`shvzk`) was a bit generic. The reason I changed the name is because the proofs used in the DKG will also be special honest verifier zero knowledge, so wanted to have a clear distinction of it.